modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/assignment/CompositeKaleoTaskAssignmentSelector.java

```diff
@@ -19,7 +19,7 @@
 /**
  * @author Michael C. Han
  */
-@Component(service = CompositeKaleoTaskAssignmentSelector.class)
+@Component(service = KaleoTaskAssignmentSelector.class)
```

Comment:
Hey @RenanUmV, it seems like a dead code. Could you please check if it is used somewhere?

Reply:
if not, just remove the class